pom.xml

```diff
@@ -2543,7 +2543,7 @@
             <dependency>
                 <groupId>org.awaitility</groupId>
```

Comment:
Shouldn't we be merging com.jayway.awaitility:awaitility & org.awaitility:awaitility ?

Reply:
WIP :)